docs/source/markdown/podman-run.1.md

```diff
@@ -1275,15 +1275,24 @@ Unset default environment variables for the container. Default environment
 variables include variables provided natively by Podman, environment variables
 configured by the image, and environment variables from containers.conf.
 
-#### **--unsetenv-all**=*true|false*
+#### **--unsetenv-all**
 
 Unset all default environment variables for the container. Default environment
 variables include variables provided natively by Podman, environment variables
 configured by the image, and environment variables from containers.conf.
 
-#### **--user**, **-u**=[_user_ | _user_:_group_ | _uid_ | _uid_:_gid_ | _user_:_gid_ | _uid_:_group_ ]
+#### **--user**, **-u**=*useropts*
 
-Sets the username or UID used and optionally the groupname or GID for the specified command.
+Sets the username or UID used and optionally the groupname or GID for the specified command:
+
+option     | description
+------     | -----------
+user       | user name
+user:group | user and group names
+uid        | user ID
+uid:gid    | user and group IDs
+user:gid   | user name with group ID
+uid:group  | user ID and group name
```

Comment:
This seems unnecessary complex, why not just say the syntax is `user[:group]` where it can be either a user or group can be specified by id or name.

Reply:
Fixed. You're right, that was stupid.